pkg/auth/manager/oidc/oidc.go

```diff
@@ -320,32 +330,41 @@ func (am *mgr) resolveUser(ctx context.Context, claims map[string]interface{}) e
 			return errtypes.PermissionDenied("no user mapping found for the given group claim(s)")
 		}
 		for _, m := range intersection {
-			username = am.oidcUsersMapping[m.(string)].Username
+			value = am.oidcUsersMapping[m.(string)].Username
 		}
+		resolve = true
+	} else if uid == 0 || gid == 0 {
+		claim = "mail"
```

Comment:
Are you sure this is `mail` and not `email` ?

Reply:
Yes https://github.com/cs3org/reva/blob/master/pkg/user/manager/ldap/ldap.go#L218, for example

Reply:
I see, thanks - what is quite misleading is that just the line after we have `email` as claim from the OIDC token just parsed. Maybe just add a comment to say this is a claim used against the user provider, otherwise it's fine.